cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -292,7 +292,17 @@ func Run(clientID string, imageName string, dataDisk bool, osID string, customTr
 		oauth, ce, gcsLogsDisabled, cloudLogsDisabled, stdoutLogsDisabled, kmsKey, kmsKeyring,
 		kmsLocation, kmsProject, noExternalIP, userLabels, storageLocation, uefiCompatible); err != nil {
 
+		postProcessErrorForNetwork(err, network, w)
+
 		return w, err
 	}
 	return w, nil
 }
+
+func postProcessErrorForNetwork(err error, network string, w *daisy.Workflow) {
+	if derr, ok := err.(daisy.DError); ok {
```

Comment:
ok -> isDaisyError. Otherwise, ok sounds like things are ok when they are not (error was thrown)

Reply:
Using `ok` is the idiomatic approach in golang: 


"use the "comma, ok" idiom to test, safely, whether the value is a string": https://golang.org/doc/effective_go.html

Reply:
And we should also call all our for indices "i"? :)

Reply:
"xxx, ok" is everywhere including standard lib. I think it's not a problem for reader to understand fluently...